src/applications/mhv/secure-messaging/tests/components/shared/AlertBackroundBox.unit.spec.jsx

```diff
@@ -5,7 +5,7 @@ import reducer from '../../../reducers';
 import AlertBackgroundBox from '../../../components/shared/AlertBackgroundBox';
 import { Alerts } from '../../../util/constants';
 
-describe('Alert Backround Box component', () => {
+xdescribe('Alert Backround Box component', () => {

```

Comment:
are we skipping this test?

Reply:
A fix is pending for this test. We are not skipping :)